Dockerfile

```diff
@@ -29,10 +29,10 @@ LABEL maintainer="Microsoft" \
 # openssh - included for ssh-keygen
 # ca-certificates
 
-# curl - required for installing jp
+# wget - required for installing jp
```

Comment:
May I know why `wget` is preferred over `curl`?

Reply:
You are right, there is no reason. I thought I had a transient error using CURL but I cannot reproduce.
I pushed a new commit to revert.